doc/modules/model_persistence.rst

```diff
@@ -36,22 +36,22 @@ persistence model, namely `pickle <https://docs.python.org/2/library/pickle.html
   0
 
 In the specific case of scikit-learn, it may be better to use
-joblib's replacement of pickle (``joblib.dump`` & ``joblib.load``),
+joblib's replacement of pickle in utils (``dump`` & ``load``),
 which is more efficient on objects that carry large numpy arrays internally as
 is often the case for fitted scikit-learn estimators, but can only pickle to the
 disk and not to a string::
 
-  >>> from sklearn.externals import joblib
-  >>> joblib.dump(clf, 'filename.joblib') # doctest: +SKIP
+  >>> from joblib import dump, load
+  >>> dump(clf, 'filename.joblib') # doctest: +SKIP
 
 Later you can load back the pickled model (possibly in another Python process)
 with::
 
-  >>> clf = joblib.load('filename.joblib') # doctest:+SKIP
+  >>> clf = load('filename.joblib') # doctest:+SKIP
 
 .. note::
 
-   ``joblib.dump`` and ``joblib.load`` functions also accept file-like object
```

Comment:
Is this even true? pickle.dump accepts a file-like, but joblib.dump doesn't.

Reply:
`joblib.dump` accepts file-like objects.